### DISCUSSION TITLE
docs/v1.12.x/advance/node-local-dns/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#2840] [Announcements]
# docs/v1.12.x/advance/node-local-dns/

https://kubeovn.github.io/docs/v1.12.x/advance/node-local-dns/

**Comment by Git4Mark** [comment]
--node-local-dns-ip=169.254.20.10
1.11.5版本添加这个参数后kube-ovn-controller启动报错，unknown flag: --node-local-dns-ip

**Reply by Git4Mark** [reply]
不好意思，没注意文档版本，忽略
